portal/factories/app.py

```diff
@@ -276,13 +276,26 @@ def configure_logging(app):  # pragma: no cover
     task_logger.addHandler(info_file_handler)
 
     # Configure Error Emails for high level log messages, only in prod mode
-    ADMINS = app.config['ERROR_SENDTO_EMAIL']
     if not app.debug:
+        ADMINS = app.config['ERROR_SENDTO_EMAIL']
+
+        creds = None
+        if app.config.get('MAIL_PASSWORD'):
+            creds = (app.config['MAIL_USERNAME'], app.config['MAIL_PASSWORD'])
```

Comment:
possible `MAIL_PASSWORD` is defined and not `MAIL_USERNAME`..

Reply:
I get the pattern though - not critical to test both when looking to see if set at all.